fix(formatter): add space after `readonly` in `TSPropertySignature`

Sysix opened the pull request:
`tasks/coverage/typescript/tests/cases/conformance/types/uniqueSymbol/uniqueSymbols.ts`

line 95:

```
interface I {
  readonly readonlyType: unique symbol;
}
```

would be formatted to:

```
interface I {
  readonlyreadonlyType: unique symbol
}
```

Dunqing commented:
### Merge activity

* **Sep 15, 1:45 AM UTC**: The merge label '0-merge' was detected. This PR will be added to the [Graphite merge queue](https://app.graphite.dev/merges?org=oxc-project&repo=oxc) once it meets the requirements.

Sysix committed:
fix(formatter): add space after `readonly` in `TSPropertySignature` (#13747)

`tasks/coverage/typescript/tests/cases/conformance/types/uniqueSymbol/uniqueSymbols.ts`

line 95:

```
interface I {
  readonly readonlyType: unique symbol;
}
```

would be formatted to:

```
interface I {
  readonlyreadonlyType: unique symbol
}
```
